shotover-proxy/benches/cassandra_benches.rs

```diff
@@ -1,22 +1,75 @@
-use cassandra_cpp::{stmt, Session};
+use cassandra_cpp::{stmt, Session, Statement};
 use criterion::{criterion_group, criterion_main, Criterion};
 use test_helpers::docker_compose::DockerCompose;
 
 #[path = "../tests/helpers/mod.rs"]
 mod helpers;
 use helpers::ShotoverManager;
 
+struct Query {
+    name: &'static str,
+    statement: Statement,
+}
+
 fn cassandra(c: &mut Criterion) {
     let mut group = c.benchmark_group("cassandra");
     group.throughput(criterion::Throughput::Elements(1));
     group.noise_threshold(0.2);
 
-    let statement =
-        stmt!("INSERT INTO benchmark_keyspace.table_1 (id, x, name) VALUES (1, 11, 'foo');");
+    let queries = [
+        Query {
+            name: "insert",
+            statement: stmt!(
+                "INSERT INTO benchmark_keyspace.table_1 (id, x, name) VALUES (1, 11, 'foo');"
+            ),
+        },
+        Query {
+            name: "select",
+            statement: stmt!("SELECT id, x, name FROM benchmark_keyspace.table_1;"),
+        },
+    ];
+    for query in queries {
```

Comment:
It's a shame we need to startup the docker containers again when this loop iterates. Any way to avoid that? This will really bump up our test runtime.

Reply:
if you compare the CI runs from this PR to another PR it bumps it up from 32mins to 34mins.
Not ideal but not world ending.

That said any ci runtime win we can get is welcome.
There are ways to improve the situation but they will be a significant investment and we should not let it stop us adding benches in the meantime.
In fact the more benches we add in the meantime will reveal how we should organize things to reduce runtime.